### PR TITLE
Create CenturyLink Cloud Pricing

### PR DESCRIPTION
**Pull request by ckleban**

### CODE REVIEW

**Comment by marycadera** [comment]
Kleban! Typo in the first line. S/b "find" not "found."

File: General/CenturyLink-Cloud-Pricing.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{{{ "title": "CenturyLink Cloud Pricing", "date": "3-10-2015", "author": "Chris Kleban", "attachments": [], "contentIsHTML": false }}}
+This page provides a quick reference on where you can found pricing information for our various cloud products.
```
Comment (nathanyoung) [inline General/CenturyLink-Cloud-Pricing.md]: "found" should be "find"

File: General/CenturyLink-Cloud-Pricing.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+{{{ "title": "CenturyLink Cloud Pricing", "date": "3-10-2015", "author": "Chris Kleban", "attachments": [], "contentIsHTML": false }}}
+This page provides a quick reference on where you can found pricing information for our various cloud products. 
+
+Q: Where can I find how much your cloud products cost?
+A: You can find pricing information for all our cloud products on this single web page.
```
Comment (nathanyoung) [inline General/CenturyLink-Cloud-Pricing.md]: The "Q" and "A" lines need to be separated by another line break.

**Comment by ckleban** [comment]
Changes made, thanks for the reviews! Ready for merging.

**Pull request merged**